docs/en/observability/slo-privileges.asciidoc

```diff
@@ -35,11 +35,12 @@ To create a role:
 Set the following privileges for the SLO Editor role:
 
 . Under *Index privileges* in the *Elasticsearch* section, add `.slo-observability-*` to the *Indices* field and `read`, `view_index_metadata`, `write`, and `manage` to the *Privileges* field.
+. Under *Cluster privileges* add `manage_transform` as SLO most likely to create the transform respect to your SLO.
```

Comment:
I'm not sure what you're trying to say here. Is there an issue or something else that explains why this privilege is needed?

Reply:
The reason of needing the privilege is that SLOs create transforms under the hood, so I would suggest something like:

> Under *Cluster privileges* add `manage_transform` as SLOs rely on Elasticsearch transforms.

Or just the need without any explanation (we are not offering explanations in the other privileges required):

> Under *Cluster privileges* add `manage_transform`.

We have recently added a new doc section called `Understanding SLO internals` with an introduction explaining the relation between SLOs and transforms. We could link to that document to provide background (https://www.elastic.co/guide/en/observability/current/slo-troubleshoot-slos.html#slo-understanding-slos).

Reply:
Users should not need `manage_transform` privileges in 8.16. We removed this requirement in an earlier version. Are you receiving reports of SLOs not working without `manage_transforms`? IF so, what version?